doc/source/serve/advanced.rst

```diff
@@ -333,3 +333,36 @@ Once a endpoint is deleted, its tag can be reused.
 
   serve.delete_endpoint("simple_endpoint")
 
+How do I call an endpoint from Python code?
+-------------------------------------------
+
+To do so, you need to get a "handle" to that endpoint. To do so,
```

Comment:
Nit: `To do so` repeated in both sentences

Reply:
thank you, fixed.